docker/files/usr/local/etc/gunicorn/joanie.py

```diff
@@ -1,5 +1,5 @@
 # Gunicorn-django settings
-bind = ["0.0.0.0:8000"]
+bind = ["0.0.0.0:8060"]
```

Comment:
I think you should leave the default 8000 port for the production server using gunicorn. Only changing the development server port should be fine.

Reply:
Ok 👍